GameData/KerbalHacks/DropTanks/Localization/en-us.cfg

```diff
@@ -1,34 +1,45 @@
-		#DROP-Wrapper-125-desc = We gathered what was left after an unfortunate Jumbo prototype explosion and thought "why waste it?". So we took a duct tape and strapped some decoupler bolts on it. Valentina named it "Wrapper", because it looked like a wrist band around a rocket. Crossfeed is switchable.
-
-		#DROP-Sleeve-125-titl = 1.25m Droptank "Sleeve"
-		#DROP-Sleeve-125-desc = The recent "Jumbo" prototype explosion was not as big as the first one, it left us with longer segments to work with. Duct tape magic, rivets and decoupler bolts presented us with a new droptank. We named it "Sleeve", is was almost certainly longer than the first prototype and closely resembled Wherner's lab coat's sleeve when wrapped around. Crossfeed is switchable.
-		
-		#DROP-Bracelet-125-titl = 1.25m Droptank "Bracelet"
-		#DROP-Bracelet-125-desc = For those who love their rockets tiny and cute. Crossfeed is switchable.
-
-		#DROP-Wristband-125-titl = 1.25m Droptank "WristBand"
-		#DROP-Wristband-125-desc = When the lack of shorter Wrapper tanks caused a riot among the rocketeers, Wherner decided to cut the current model in half. Crossfeed is switchable.
-	}
+// en-us.cfg v1.1.0.2
+// Drop Tanks (DROP)
+// created: 19 Dec 2021
+// updated: 22 Jan 2023
+
+// THIS FILE:
+//   CC BY-NC-SA 4.0 by zer0Kerbal
+//   generated by [zer0Kerbal](https://github.com/zer0Kerbal)
+
+Localization
+{
+	en-us
+	{
+	// Addon specific
+		#DROP-mod-name = Drop Tanks
+		#DROP-tags = drop tank wrap enceos
+	
+	// Parts
+		#DROP-Cap-125-titl = 1.25m Droptank Aero Cap
+		#DROP-Cap-125-desc = After many requests to hide the unsightly end of the Wrapper tanks from the airflow our engineering division devised this beautiful cap. We even managed to fit a small fuel canister inside.
+
+		#DROP-Wrapper-125-titl = 1.25m Droptank "Wrapper"
+		#DROP-Wrapper-125-desc = We gathered what was left after an unfortunate Jumbo prototype explosion and thought "why waste it?". So we took a duct tape and strapped some decoupler bolts on it. Valentina named it "Wrapper", because it looked like a wrist band around a rocket. Crossfeed is switchable.
+
+		#DROP-Sleeve-125-titl = 1.25m Droptank "Sleeve"
+		#DROP-Sleeve-125-desc = The recent "Jumbo" prototype explosion was not as big as the first one, it left us with longer segments to work with. Duct tape magic, rivets and decoupler bolts presented us with a new droptank. We named it "Sleeve", is was almost certainly longer than the first prototype and closely resembled Wherner's lab coat's sleeve when wrapped around. Crossfeed is switchable.
+		
+		#DROP-Bracelet-125-titl = 1.25m Droptank "Bracelet"
+		#DROP-Bracelet-125-desc = For those who love their rockets tiny and cute. Crossfeed is switchable.
+
+		#DROP-Wristband-125-titl = 1.25m Droptank "WristBand"
+		#DROP-Wristband-125-desc = When the lack of shorter Wrapper tanks caused a riot among the rocketeers, Wherner decided to cut the current model in half. Crossfeed is switchable.
+
+	// B9partswitch

```

Comment:
nice! appreciate this detail. 👍 🦃 🧧 
I will scrape the stock dictionary to use stock strings for as much as possible and update the B9 patch.